modules/sum-transformers/additional/summary/summary_params.go

```diff
@@ -0,0 +1,20 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package summary
+
+type Params struct {
```

Comment:
I am wondering why are not you simply using []string instead of Params?

Reply:
Currently, we have only **properties** as the parameters. But there may be cases we need to add more parameters like **certainty**, and **limit** as in the ner-transformers module. That's the reason for using a struct